hyosin-Jang/230411/과제진행하기.js

```diff
@@ -0,0 +1,60 @@
+function solution(plans) {
+  waiting = [];
+  finish = [];
+
+  // 시간 => 분으로 통일
+  plans = plans.map((plan) => getMinute(plan));
+
+  plans.sort((a, b) => a[1] - b[1]); // 오름차순 정렬
+  let curSubject = plans.shift();
+
+  while (1) {
+    if (plans.length === 0) break;
```

Comment:
```suggestion
  while (plans.length) {
```
로 한줄이라도 줄여볼 수 있을 것 같아요!

Reply:
`while(1)`을 습관처럼 적는데 말씀대로 조건문에 종료조건을 적으면 코드를 줄일 수 있어서 좋은거 같아요! 수정해보겠습니당 👍